teps/0086-changing-the-way-result-parameters-are-stored.md

```diff
@@ -249,7 +251,7 @@ storing arbitrarily large data.
 
 In this option we introduce a way to pass one TaskRun’s output results into another TaskRun’s input parameters by reference, instead of
 only by value as they are today.
-The `byRef` could also be configured and handled by the controller and not something that needs to be made explicit at authoring time.
+The result `type` could also be configured as `stringReference` or `arrayReference` and handled by the controller and not something that needs to be made explicit at authoring time.
```

Comment:
- will the `objectReference` be supported too after TEP-0075 is implemented?

- could you clarify what "handled by the controller and not something that needs to be made explicit at authoring time" means?

Reply:
- Yes we can add `objectReference` when TEP-0075 is implemented.
- The original proposal was assuming the reference part can be determined by the controller implicitly. However, since Tekton 0.36, the result `type` field is not become required. Let me update the statement to make it as an explicit field.